.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--- Provide a general summary of your changes in the title. -->
+
+## Description
+
+<!--- Use this section to describe your changes. If your changes required
+testing, include information about the testing environment and the tests you
+ran. If your test fixes a bug reported in an issue, don't forget to include the
+issue number. If your PR is still a work in progress, that's totally fine – just
+include a note to let us know. -->
+
+### Types of change
+
+<!-- What type of change does your PR cover? Is it a bug fix, an enhancement
+or new feature, or a change to the documentation? -->
+
+## Checklist
+
+<!--- Before you submit the PR, go over this checklist and make sure you can
+tick off all the boxes. [] -> [x] -->
+
+- [ ] I confirm that I have the right to submit this contribution under the project's MIT license.
+- [ ] I ran the tests, and the `update` scripts in the `.github` folder.
```

Comment:
I'd have these as two separate boxes, because you typically run the tests before opening the PR, but the "update" script will need to be called right before merging.

Actually, should we have two subsections to make this distinction more clear or is that too much?

Reply:
Separating them into two checkboxes makes sense, yeah. But what do you mean with subsections?

Reply:
Never mind that bit 🙈